inspirehep/modules/workflows/utils/__init__.py

```diff
@@ -248,3 +251,43 @@ def insert_wf_record_source(json, record_uuid, source):
     else:
         record_source.json = json
     db.session.commit()
+
+
+def start_workflow(workflow, data=None, workflow_object=None):
+    """Start a workflow creating any needed models.
+
+    Due to the task `invenio_workflows.start` being a celery task itself, it
+    delays to it's own execution the creation of the
+    `invenio_workflows.Workflow` db entry, rendering the task unretriable if
+    for any reason it gets stuck before that step (for example, if rabbitmq
+    flushes the queue, or if a worker acks the task without actually doing
+    anything).
+
+    Params:
```

Comment:
Isn't it `Arguments` in Napoleon?

Reply:
I think it does not matter, but the example uses `Args:`